source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -302,6 +302,20 @@ MongoClient from the one used for test operations. For example::
             replSetStepDown: 20
             force: false
 
+**Note:** The "replSetStepDown" command often fails with the following
+transient error (see `SERVER-48154`_)::
+
+  {
+    "ok" : 0,
+    "errmsg" : "Unable to acquire X lock on '{4611686018427387905: ReplicationStateTransition, 1}' within 1000ms. opId: 922, op: conn30, connId: 30.",
```

Comment:
Here too, I don't think the `1000ms` would appear if the limit is 20s in the above example.

However, the rediscover-quickly-after-step-down.yml file uses `1000ms`. Perhaps the example should be updated to match that for consistency?

Reply:
Updated runAdminCommand example to match what's used in rediscover-quickly-after-step-down.yml.